src/components/layout/AddToWishlistButton/AddToWishlistButton.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import Button from '@mui/material/Button';
+import axios from 'axios';
+import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
+import { useNavigate } from 'react-router-dom';
+
+const AddToWishlistButton = ({ album }) => {
+  const [isAdded, setIsAdded] = useState(false);
+  const { status } = useAuth();
+  const navigate = useNavigate();
+  const isUserLoggedIn = status === AuthStatus.LoggedIn;
+
+  const fetchOrCreateWishlist = async () => {
+    try {
+      const response = await axios.post(
+        'http://localhost:8000/api/v1/wishlist',
+        { albumId: album._id },
```

Comment:
I'm reviewing Yelena's ticket that adds this endpoint and discussing if we need to pass in an album ID. Since the frontend is already going to make a subsequent call to add items to the wishlist via the `PATCH` endpoint, I'm not sure if it makes sense to accept an album ID on the create wishlist endpoint.

One thing I think we should do (time permiting) is have the backend create and return an empty list when the user is first created/registered or attach their existing wish list when they register. The app can then store that in redux/localstorage and use it throughout without having to necessarily make this fetchOrCreateWishlist call from within every `Add To Wishlist` button that appears on the page. 

For now though, let's keep this call, just without passing in the album id since the backend is going to ignore that anyway.

Reply:
Done.